CannibalisticZombies/Assets/Scripts/UI/InventoryUI.cs

```diff
@@ -0,0 +1,155 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+namespace CannibalisticZombies 
+{
+    public class InventoryUI : MonoBehaviour
+    {
+        public Inventory inventory;
```

Comment:
I do not think we would want to access all of these methods publicly, as all inventory shenanigans relating to UI should be done in this script. For all the variables that this applies to below (Not including `LINELENGTH`), lets use `[SerializeField] private .........`. We do not want to mark things public unless they are to be accessed by other scripts.